large_language_model/gpt3_blend.sh

```diff
@@ -10,5 +10,5 @@ C4_5="${COM_DIR}/c4_en_5_c4_spm_text_document"
 C4_6="${COM_DIR}/c4_en_6_c4_spm_text_document"
```

Comment:
Can you remove c4_en_0_c4_spm_text_document to c4_en_5_c4_spm_text_document since that is not going to be a part of the benchmarking region?
Also update the DATA BLEND

Reply:
Also reflect this in the README